doc/ko: Add missing options in uftrace-replay.md

sinsohi opened the pull request:
Identified and corrected discrepancies between the English and
Korean versions of uftrace-replay.md:

1. The description of the --disable option in the Korean version
   differed from the English version. This has been correctly
   retranslated.

2. Added the missing options --trace=*STATE*, --srcline, and
   --no-args to the Korean version.

3. Included the 'task' and 'module' fields for the
   -f *FIELD*, --output-fields=*FIELD* option in the Korean version,
   which were previously omitted.

These changes bring the Korean documentation in line with the
English version, ensuring consistency and completeness.

Signed-off-by: Sin Sohi <sohi8451@naver.com>

honggyukim commented on doc/ko/uftrace-replay.md:
```diff
@@ -110,8 +116,11 @@ REPLAY 옵션
+:   더 이상 사용되지 않는다. 대신 `--trace=off`를 사용한다.
+
+\--trace=*STATE*
+:   uftrace 트레이싱 상태를 설정한다. 가능한 상태는 `on`과 `off`이다. 기본값은
+`on`이다. 이는 `trace_on` 트리거나 에이전트와 함께 사용할 때만 의미가 있다.
```
How about making it as follows?
```suggestion
:   uftrace 트레이싱 상태를 설정한다. 가능한 상태는 `on`과 `off` 이며 기본값은
   `on`이다. 이는 `trace_on` 트리거나 에이전트와 함께 사용할 때만 의미가 있다.
```

honggyukim reviewed (commented):
Thanks for the patch. I've just left one minor comment and others are fine.

But could you please enhance your commit message? I've asked many times about rewriting commit messages in various PRs so you can get some ideas.

The most recent request can be found at https://github.com/namhyung/uftrace/pull/1946#pullrequestreview-2240215035. Please read the comment and other comments below so that you can get more ideas what I have been asking before merge a PR.

I can see your commit message at https://github.com/namhyung/uftrace/pull/1957.patch.

It would be helpful if @gichoel or @ParkSeungHyeok can help him write better commit messages.

gichoel commented:
First, thanks for your contribution.

However, there are some rules for writing a good readable commit message, and the commit message improvement that `honggyukim` is talking about seems to be related to the **50/72 rule[1]**.

In the case of the commit message currently written, the subject does not exceed 50 characters, so it satisfies the rule, but the body does not satisfies the rule because it is not broken every 72 characters.

Could you please make this change?


[1] https://www.gitkraken.com/learn/git/best-practices/git-commit-message#git-commit-message-structure

sinsohi commented:
Thank you for your feedback. 
I've updated the commit message and code as requested:

1. Improved Korean translation for the --trace=*STATE* option.
2. Adjusted the commit message format following the 50/72 rule.

Let me know if any further modifications are needed.

honggyukim commented:
Please update your name correctly as I mentioned at https://github.com/namhyung/uftrace/pull/1957#pullrequestreview-2277497840.
> The most recent request can be found at https://github.com/namhyung/uftrace/pull/1946#pullrequestreview-2240215035.

namhyung commented:
Please don't merge, just rebase your changes on top of the current master.

sinsohi commented:
Thank you for your feedback.
I have reflected your requirements.
Please check it out.

sinsohi commented:
I'm having trouble resolving this. Would it be okay if I delete this PR and submit a new one

honggyukim commented:
Please update it here and don't open a new PR. If you're a contribution academy mentee, then you can ask @gichoel.

sinsohi commented:
I apologize. I didn't check your response and, due to severe errors, I hastily closed the PR and deleted the repository. I'm really sorry, but would it be alright if I fork again and resend the PR?

sinsohi commented:
I apologize. I didn't check your response and, due to severe errors, I hastily closed the PR and deleted the repository. I'm really sorry, but would it be alright if I fork again and resend the PR?

honggyukim commented:
You don't have to say sorry. We know how it is difficult at the first try. The issue is that I am quite busy these days so cannot explain all the details.

I just hope someone else can help you with this issue because helping new contributors is also a great contribution to the project.

gichoel commented:
`sinsohi` is not a Contribution Academy mentee, so I am limited in helping directly, but I'd like to help new contributors get acclimated to uftrace.

However, after checking the commit hash of the PR and the commit hash of `sinsohi`'s forked repository, it seems that the repository was already deleted, recreated, and then the commits were uploaded again.

@honggyukim 
From what I have investigated, it seems that the PR cannot be reopened in this case, so would it be correct to create a new PR and then mention the this PR?

honggyukim commented:
Sorry @gichoel. I misunderstood that.

@sinsohi Do you mind if I modify your patch rather than asking many requests? If you're okay, then I can just merge it into the master directly without creating a new PR.

sinsohi commented:
I would be really grateful if you could do that.

honggyukim commented:
Thanks. I will do that when I have some time.

sinsohi commented:
@honggyukim Please write my name as Sin Sohi. Thank you.

honggyukim commented:
@sinsohi Sorry, I've been busy for a while. I will do that within a few days.

honggyukim commented:
Hi @sinsohi, sorry for making delay. I've applied your patch at 5430d02d2d3ce2478e7679bbf35f8c462f22629a and it's now in master. Thanks for your work!
